app/templates/components/forms/wizard/attendee-step.hbs

```diff
@@ -0,0 +1,115 @@
+                <th class="center aligned">
+                  {{t 'Include'}}
+                </th>
+                <th class="center aligned">
+                  {{t 'Require'}}
+                </th>
+              {{/if}}
+            </tr>
+          </thead>
+          <tbody>
+            {{#each data.customForms as |field|}}
+              <tr class="{{if field.isIncluded 'positive'}}">
+                <td class="{{if device.isMobile 'center' 'right'}} aligned">
+                  <label class="{{if field.isFixed 'required'}}">
+                    {{field.name}}
+                  </label>
+                </td>
+                <td class="center aligned">
+                  {{ui-checkbox class='slider'
+                                checked=field.isIncluded
+                                disabled=field.isFixed
+                                onChange=(action (mut field.isIncluded))
+                                label=(if device.isMobile (t 'Include'))}}
+                </td>
+                <td class="center aligned">
+                  {{ui-checkbox class='slider'
+                                checked=field.isRequired
+                                disabled=field.isFixed
+                                onChange=(action (mut field.isRequired))
+                                label=(if device.isMobile (t 'Require'))}}
+                </td>
+              </tr>
+            {{/each}}
+          </tbody>
+        </table>
+      </div>
+    </div>
+    <h3 class="ui dividing header">
+      <i class="options box icon"></i>
+      <div class="content">
+        {{t 'Registration Options'}}
+      </div>
+    </h3>
+    <div class="field">
+      <label>{{t 'REGISTRATION TIME LIMIT'}}</label>
+      <div class="{{unless device.isMobile 'two wide'}} field">
+        {{input type='number' id='orderExpiryTime' value=data.event.orderExpiryTime min="1" max="60" step="1"}}
+      </div>
+    </div>
+  {{/if}}
+  <div class="spacer-50"></div>
+
+  <div class="{{if device.isMobile 'mini four' 'right floated large'}} ui fields buttons">
+    <button class="ui three field left labeled icon button {{if isLoading 'disabled'}}" type="button" {{action 'move' 'backwards'}}>
+      {{t 'Previous'}}
+      <i class="left chevron icon"></i>
+    </button>
+    <button class="ui three field right labeled icon button {{if isLoading 'disabled'}}" type="button" {{action 'move' 'forwards'}}>
+      {{t 'Forward'}}
+      <i class="right chevron icon"></i>
+    </button>
+    <button class="blue ui three field right labeled icon button {{if isLoading 'disabled'}}" type="button" {{action 'saveDraft'}}>
+      {{t 'Save draft'}}
+      <i class="save icon"></i>
+    </button>
+    {{#if data.event.locationName}}
+      <button class="green ui three field right labeled icon button {{if isLoading 'disabled'}}" type="button" {{action 'publish'}}>
+        {{t 'Publish'}}
+        <i class="check icon"></i>
+      </button>
+    {{/if}}
+  </div>
+
```

Comment:
A little less priority but I guess we can get rid of the extra line at Line 114 and 93.